ports/libxml2/portfile.cmake

```diff
@@ -1,5 +1,4 @@
-vcpkg_from_gitlab(
-    GITLAB_URL https://gitlab.gnome.org/
+vcpkg_from_github(
```

Comment:
Is this change expected on this PR?

Reply:
I had it in all three. It should get dropped on merge since #4304 already merged the commit.

Reply:
Removed after a rebase.